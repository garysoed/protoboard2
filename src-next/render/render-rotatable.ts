import {OperatorFunction} from 'rxjs';
import {map} from 'rxjs/operators';


export function renderRotatable(): OperatorFunction<number|undefined, string> {
  return map(rotationDeg => `rotateZ(${rotationDeg ?? 0}deg)`);
}
