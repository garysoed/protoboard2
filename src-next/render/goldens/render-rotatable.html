<pbt-test><gst-fake-shadow><div id="container" style="height: 50px; width: 50px; transform: rotateZ(123deg);">PB</div></gst-fake-shadow></pbt-test>